requirements.txt

```diff
@@ -83,10 +83,8 @@ psycopg2==2.9.6
 pycparser==2.21
 pydantic==1.10.8
 pydantic-sqlalchemy==0.0.9
-pyee==8.2.2
```

Comment:
why removed pyee & pyppeteer?

Reply:
These are dependency of playwright, it doesn't needed to be added separately